lib/uo_cli.rb

```diff
@@ -1578,7 +1578,7 @@ def self.install_python_dependencies
       end
 
       if @opthash.subopts[:feature]
-        if !@opthash.subopts[:feature].to_s.include?('.json')
+        if !@opthash.subopts[:feature].to_s.end_with?('json')
```

Comment:
I could replace `if !` with `unless` to use some ruby-specific syntax. This way is perhaps a little more universal, and using `unless` is perhaps a little more specific/clear about what it's doing? Do you care which we use?

Reply:
whichever you prefer!